src/simple-map.js

```diff
@@ -0,0 +1,20 @@
+export default class SimpleMap {
```

Comment:
Please run `npm run lint` and ensure that changes do not introduce new style errors.